eng/Versions.props

```diff
@@ -72,6 +72,14 @@
     <MicrosoftExtensionsHttpVersion>9.0.3</MicrosoftExtensionsHttpVersion>
     <SystemFormatsAsn1Version>9.0.3</SystemFormatsAsn1Version>
     <SystemTextJsonVersion>9.0.3</SystemTextJsonVersion>
+    <!-- OpenTelemetry (OTel) -->
+    <OpenTelemetryVersion>1.11.2</OpenTelemetryVersion>
+    <OpenTelemetryInstrumentationAspNetCoreVersion>1.11.1</OpenTelemetryInstrumentationAspNetCoreVersion>
+    <OpenTelemetryInstrumentationHttpVersion>1.11.1</OpenTelemetryInstrumentationHttpVersion>
+    <OpenTelemetryInstrumentationExtensionsHostingVersion>1.11.2</OpenTelemetryInstrumentationExtensionsHostingVersion>
+    <OpenTelemetryInstrumentationRuntimeVersion>1.11.1</OpenTelemetryInstrumentationRuntimeVersion>
+    <OpenTelemetryInstrumentationGrpcNetClientVersion>1.11.0-beta.2</OpenTelemetryInstrumentationGrpcNetClientVersion>
```

Comment:
I don't see this property being used. We shouldn't use it in our templates since it isn't stable.